src/lxc/conf.c

```diff
@@ -3469,8 +3469,10 @@ int lxc_map_ids(struct lxc_list *idmap, pid_t pid)
 	 * will protected it by preventing another user from being handed the
 	 * range by shadow.
 	 */
+	#if HAVE_LIBCAP
 	uidmap = idmaptool_on_path_and_privileged("newuidmap", CAP_SETUID);
 	gidmap = idmaptool_on_path_and_privileged("newgidmap", CAP_SETGID);
+	#endif
```

Comment:
Sorry, you're right, I will close this PR. I found the issue on lxc-2.0.8 and I didn't see that the bug was fixed by bc5b27d6f6d166d2a6df47982cbe36041ce6b735.